test_data/yast/lvm_thin_provisioning/lvm_thin_provisioning_uefi.yaml

```diff
@@ -0,0 +1,38 @@
+disks:
+  - name: vda
+    partitions:
+      - size: 300mb
+        role: raw-volume
+        id: efi
+        formatting_options:
+          should_format: 1
+          filesystem: fat
+        mounting_options:
+          should_mount: 1
+          mount_point: /boot/efi
+      - size: 15GiB
+        role: raw-volume
+        id: linux-lvm
+      - role: raw-volume
+        id: linux-lvm
+lvm:
+  volume_groups:
+  - name: vg-no-thin
+    devices:
+      - /dev/vda2
+    logical_volumes:
+      - name: lv-swap
+        size: 2GiB
+        role: swap
+      - name: lv-root
+        role: operating-system
+  - name: vg-thin
+    devices:
+      - /dev/vda3
+    logical_volumes:
+      - name: lv-thin-pool
+        volume_type: thin_pool
+      - name: lv-home
+        volume_type: thin_volume
```

Comment:
what about just name it 'type' ? we are already in section 'logical volumes' unless there is some other type, should be enough.

Reply:
I am a bit skeptical, because in the partitioner function, it will be $lv->{type} which is not as clear as the current naming. WDYT?

Reply:
lv is always 'logical volume' as 'pv' is physical volume ,etc. so that is the 'type of .. the thing' I don't see any issue with that.

Reply:
:hushed: done